trackintel/preprocessing/util.py

```diff
@@ -59,3 +64,35 @@ def calc_temp_overlap(start_1, end_1, start_2, end_2):
         overlap_ratio = temp_overlap / dur.total_seconds()
 
     return overlap_ratio
+
+
+def applyParallel(dfGrouped, func, n_jobs, print_progress, **kwargs):
+    """
+    Funtion warpper to parallize funtions after .groupby().
+
+    Parameters
+    ----------
+    dfGrouped: pd.DataFrameGroupBy
+        The groupby object after calling df.groupby("user_id").
```

Comment:
I think we can leave the description like this but it should work with any groupby object, shouldn't it?

Reply:
I changed this from `df.groupby("user_id")` into `df.groupby(COLUMN)`